gulpfile.js

```diff
@@ -91,37 +91,47 @@ function writeServiceWorkerFile() {
     cacheId: 'polymerproject',
     staticFileGlobs: [
       `${rootDir}/1.0/index.html`,
```

Comment:
These three (index, shell, about) shouldn't be in the static file list, since they're assembled on the server.